pkg/protocol/header.go

```diff
@@ -125,6 +126,8 @@ type ResponseHeader struct {
 	h     []argsKV
 	bufKV argsKV
 
+	Trailer Trailer
```

Comment:
Can we make it the same format
<img width="415" alt="image" src="https://user-images.githubusercontent.com/37136584/211781809-25d6d746-8852-4f57-be38-a65e0edd3e71.png">